src/clients/users_finish.rb

```diff
@@ -71,12 +73,20 @@ def main
 
           # During installation, some package could add a new user, so we
           # need to read them again before writing.
-          Users.SetExportAll(true)
+
+          # 1. Export users imported in inst_autosetup (and store them)
+          Users.SetExportAll(false)
           saved = Users.Export
-          Users.ReadLocal
+          log.info("Users to import: #{saved}")
+
+          # 2. Read users and settings from the installed system
+          # (bsc#965852, bsc#973639, bsc#974220 and bsc#971804)
+          Users.Read
+
+          # 3. Import users exported in step 1
```

Comment:
It's actually `Merge users from the system with new users from AutoYast profile` ?